quadrat/template-parts/header.php

```diff
@@ -14,7 +14,8 @@
 <header class="site-header" role="banner">
 	<?php
 	echo do_blocks(
-		'<!-- wp:group {"style":{"spacing":{"padding":{"right":"35px","left":"35px"}}}} --><div class="wp-block-group" style="padding-right:35px;padding-left:35px">
+		'<!-- wp:group {"style":{"spacing":{"padding":{"top":"35px","right":"35px","left":"35px"}}}} --><div class="wp-block-group" style="padding-top:35px;padding-right:35px;padding-left:35px">
```

Comment:
What do you think about using CSS to create the padding for the header instead?

If we add the padding via a block style attribute (like is being done here), it can in theory be controlled by the site editor. The drawback is this value isn't responsive. And since this is already a work around for the nav, this template part wouldn't actually be consumed by the site editor when the theme switches to using it.

Reply:
> And since this is already a work around for the nav, this template part wouldn't actually be consumed by the site editor when the theme switches to using it.

I think this will not be the case when the Site Editor launches - we should be able to use the location attribute and switch to a block template.

Reply:
> The drawback is this value isn't responsive. 

Could we think about this differently by follow the intrinsic design idea?